### DISCUSSION TITLE
Folders for artists videos

### DISCUSSION THREAD

**Discussion by KINGLIFER** [RandomNinjaAtk/arr-scripts#60] [Q&A] [answered]
Is there a way to place videos into folders? 
example of a completed video
/musicvideo/Donna Summer/Bad Girls (1979).mkv
or
/musicvideo/Donna Summer/Bad Girls.mkv

I would not mind if the only way would be to have the videos in the music folders nested in the albums. I can use software like everything or BRU to fix them afterwards.

If there isn't a method or switch in config will there be a in progress plan to implement one?

Thank you.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Videos are already in artist folders.... So if thats what you're referring too, it already does that...

**Reply by RandomNinjaAtk** [reply]
Can’t review anything without logs.

also post your config

**Reply by RandomNinjaAtk** [reply]
Your path for videos is wrong based on the container config you posted above. Should be /videos and not /data/videos….


Helps to configure things correctly. Honestly don’t have time to look at it any further at the moment
